storyscript/hub/sdk/service/CommandArgument.py

```diff
@@ -0,0 +1,29 @@
+class Argument:
```

Comment:
```suggestion
class CommandArgument:
```

(to match the file name)